docs/openapi/delivery/pn-delivery.yaml

```diff
@@ -1,1085 +1,1260 @@
-          in: path
-          name: iun
-          required: true
+      - name: iun
+        in: path
+        description: Identificativo Univoco Notifica
+        required: true
+        schema:
           type: string
-        - description: 'Customer taxId, used in B2B AppIO.'
-          in: header
+      - name: docIdx
+        in: path
+        description: indice del documento nella lista partendo da 0.
+        required: true
+        schema:
+          type: number
+      - name: x-pagopa-cx-taxid
+        in: header
+        description: "Customer taxId, used in B2B AppIO."
+        required: true
+        schema:
           maxLength: 16
           minLength: 16
-          name: x-pagopa-cx-taxid
-          pattern: '^[A-Z]{6}[0-9]{2}[A-Z][0-9]{2}[A-Z][0-9]{3}[A-Z]$'
-          required: true
+          pattern: "^[A-Z]{6}[0-9LMNPQRSTUV]{2}[A-Z]{1}[0-9LMNPQRSTUV]{2}[A-Z]{1}[0-9LMNPQRSTUV]{3}[A-Z]{1}"
           type: string
       responses:
-        '200':
+        "200":
           description: OK
-          schema:
-            $ref: '#/definitions/FullReceivedNotification'
-        '400':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/NotificationAttachmentDownloadMetadataResponse'
+        "400":
           description: Bad request
-          schema:
-            $ref: '#/definitions/Problem'
-      tags:
-        - AppIO-PN-Notification
-      description: >-
-        Utilizzato da Persone Fisiche e Persone Giuridiche per accedere ai
-        dettagli delle  notifiche ricevute.
-      operationId: getReceivedNotification
-      summary: Accesso notifiche ricevute
-  '/delivery/notifications/sent/{iun}/attachments/documents/{docIdx}':
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+  delivery-push/{iun}/legal-facts/{legalFactType}/{legalFactId}:
     get:
-      produces:
-        - application/json
+      tags:
+      - AppIO-PN-LegalFacts
+      summary: Download atto opponibile a terzi
+      description: Permette di scaricare un atto opponibile a terzi
+      operationId: getLegalFact
       parameters:
-        - description: Identificativo Univoco Notifica
-          in: path
-          name: iun
-          required: true
-          type: string
-        - in: path
-          name: docIdx
-          required: true
-          type: number
-        - description: 'Customer taxId, used in B2B AppIO.'
-          in: header
+      - name: iun
+        in: path
+        description: Identificativo Univoco Notifica
+        required: true
+        schema:
+          type: string
+      - name: legalFactType
+        in: path
+        description: Categoria dell'atto opponbile a terzi
+        required: true
+        style: simple
+        explode: false
+        schema:
+          title: Tipi di atti opponibili a terzi
+          type: string
```

Comment:
imho the type `enum` in the old spec was better here instead of `string`. in the mock code what should we do?
this needs attention